app/src/main/java/io/intrepid/contest/screens/entrysubmission/entryimage/EntryImagePresenter.java

```diff
@@ -39,7 +43,28 @@ public void onBitmapReceived(Bitmap bitmap) {
 
     @Override
     public void onEntrySubmitted() {
-        Timber.d("Submit entry");
+        Timber.d("Entry creation API call.");
+
+        String contestId = persistentSettings.getCurrentContestId().toString();
+        EntryRequest entryRequest = new EntryRequest(entryName, bitmap);
+
+        Disposable disposable = restApi.createEntry(contestId, entryRequest)
+                .compose(subscribeOnIoObserveOnUi())
+                .subscribe(response -> showResult(response), throwable -> {
+                    Timber.d("API error: " + throwable.getMessage());
+                    view.showApiErrorMessage();
```

Comment:
Ohh I thought it would mess up tests. But I guess I also have android bitmaps and other "androidy things" that just happen to be there.

So to @abassawo 's [point in PR #14](https://github.com/IntrepidPursuits/ContestAndroid/pull/14#discussion_r98938230), I could have a single view method that takes in strings for message/title and decides how to display it?